mod/wet4/views/default/page/elements/head.php

```diff
@@ -66,13 +66,14 @@
     }
 
   }
+
 } else {
 
   $current_url = explode('/', $_SERVER['REQUEST_URI']);
   $page_entity = get_entity($current_url[count($current_url) - 1]);
 
   if ($elgg_entity instanceof ElggUser || $elgg_entity instanceof ElggGroup) {
```

Comment:
The current `OR`'d condition `if ($elgg_entity instanceof ElggUser || $elgg_entity instanceof ElggGroup) {` since it contains a ternary with the same terms inside.
EX:
```
if (cond1 || cond2) {
  result = cond1 ? value1 : value2
}
...
```
Is the same as
```
if (cond1) {
  result = value1
}
else if (cond2) {
  result = value2
}
```
Unless i'm missreading it